subsys/caf/modules/Kconfig.ble_adv

```diff
@@ -35,6 +35,73 @@ config CAF_BLE_ADV_DIRECT_ADV
 config CAF_BLE_ADV_FAST_ADV
 	bool "Enable period of fast advertising"
 
+config CAF_BLE_ADV_CUSTOM_INT
+	bool "Enable custom advertising intervals"
+
+if CAF_BLE_ADV_CUSTOM_INT
+
+config CAF_BLE_ADV_FAST_INT_MIN_1
+	hex "Minimum Advertising Interval in faster mode (N * 0.625 milliseconds)"
```

Comment:
in faster mode -> during fast advertising
in slower mode -> during slow advertising
To be more consistent with the documentation (it would be easier to understand for users I think)

Reply:
We could also call Kconfig options `CONFIG_CAF_BLE_ADV_FAST_INT_MIN/MAX`, `CONFIG_CAF_BLE_ADV_SLOW_INT_MIN/MAX` and the Fast intervals configuration should depend on enabling `CONFIG_CAF_BLE_ADV_FAST_ADV`